channels/management/commands/runserver.py

```diff
@@ -26,11 +26,14 @@ def add_arguments(self, parser):
             help='Run the old WSGI-based runserver rather than the ASGI-based one')
         parser.add_argument('--http_timeout', action='store', dest='http_timeout', type=int, default=60,
             help='Specify the daphane http_timeout interval in seconds (default: 60)')
+        parser.add_argument('--websocket_handshake_timeout', action='store', dest='websocket_handshake_timeout', type=int, default=5,
+            help='Specify the daphane websocket_handshake_timeout interval in seconds (default: 5)')
```

Comment:
Ah good catch, seems I copied and pasted it from the parameter above, so I fixed that one too.